### DISCUSSION TITLE
Detect hardware back button press when using only capacitor (without ionic)

### DISCUSSION THREAD

**Discussion by iarkaroy** [ionic-team/capacitor#3399] [Q&A] [answered]
I have added capacitor to my existing vue app. Looking for a way to detect hardware back button to close the app. Tried adding event listener for `ionBackButton` but it is not working. I suppose that event is fired in an ionic app (which it is not). Can anyone please guide me how can I detect hardware back button? Please note that I did not initialised an ionic app from scratch. I have added the capacitor cli, core and platforms (android and ios) on top of existing vue app.

Thanks

**Comment by imhoffd** [comment, marked as answer]
I think you're looking for the `backButton` event in the App plugin: https://capacitorjs.com/docs/apis/app#method-addListener-3

**Reply by iarkaroy** [reply]
Yes, this is what I was looking for. Thanks.
